cdap-docs/admin-manual/source/installation/cloudera.rst

```diff
@@ -109,19 +108,55 @@ Downloading and Distributing Packages
 
 Downloading and Installing CSD
 ------------------------------
-- Download the CDAP CSD (Custom Service Descriptor) by `downloading the JAR file 
-  <http://cask.co/resources/#cdap-integrations>`__.
+To install CDAP on a cluster managed by Cloudera, we have available a Custom Service
+Descriptor (CSD) which you can install onto your CM server. This adds CDAP to the list of
+available services which CM can install. 
+
+.. _cloudera-compatibility-matrix:
+
++---------------------------------------------------------------------------------------+
+| Supported Cloudera Manager (CM) and Cloudera Data Hub (CDH) Distributions             |
++------------+-----------------------+---------------------+----------------------------+
+| CM Version | CDH Version           | CSD Version         |  CDAP Parcel Version       |
++============+=======================+=====================+============================+
+| 5.5        | 5.5                   | 3.3.x               | *Matching CSD major.minor* | 
++------------+-----------------------+---------------------+----------------------------+
+| 5.5        | 5.4.x                 | 3.1.x through 3.3.x | *Matching CSD major.minor* |
++------------+-----------------------+---------------------+----------------------------+
+| 5.5        | no greater than 5.3.x | 3.0.x through 3.3.x | *Matching CSD major.minor* |
```

Comment:
minor suggestion: the matrix now looks a bit intimidating at first.  is there any shading or separation we can add to make it more readable?  ie, the 3 CM 5.5 rows shaded similarly, then the 2 CM 5.4 rows, etc...  just a thought.